[2025-03-08T20:27:51.352+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: test_dag.print_date manual__2025-03-08T19:51:24.118312+00:00 [queued]>
[2025-03-08T20:27:51.359+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: test_dag.print_date manual__2025-03-08T19:51:24.118312+00:00 [queued]>
[2025-03-08T20:27:51.360+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2025-03-08T20:27:51.360+0000] {taskinstance.py:1284} INFO - Starting attempt 8 of 9
[2025-03-08T20:27:51.360+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2025-03-08T20:27:51.373+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): print_date> on 2025-03-08 19:51:24.118312+00:00
[2025-03-08T20:27:51.379+0000] {standard_task_runner.py:55} INFO - Started process 83 to run task
[2025-03-08T20:27:51.382+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'print_date', 'manual__2025-03-08T19:51:24.118312+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/__pycache__/test_dag_ai.py', '--cfg-path', '/tmp/tmpl946xo7c']
[2025-03-08T20:27:51.383+0000] {standard_task_runner.py:83} INFO - Job 72: Subtask print_date
[2025-03-08T20:27:51.394+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-03-08T20:27:51.430+0000] {task_command.py:389} INFO - Running <TaskInstance: test_dag.print_date manual__2025-03-08T19:51:24.118312+00:00 [running]> on host d781a364bdf5
[2025-03-08T20:27:51.501+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_dag
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2025-03-08T19:51:24.118312+00:00
AIRFLOW_CTX_TRY_NUMBER=8
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-03-08T19:51:24.118312+00:00
[2025-03-08T20:27:51.504+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2025-03-08T20:27:51.505+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'date']
[2025-03-08T20:27:51.516+0000] {subprocess.py:86} INFO - Output:
[2025-03-08T20:27:51.571+0000] {subprocess.py:93} INFO - Sat Mar  8 20:27:51 UTC 2025
[2025-03-08T20:27:51.571+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-08T20:27:51.598+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=print_date, execution_date=20250308T195124, start_date=20250308T202751, end_date=20250308T202751
[2025-03-08T20:27:53.385+0000] {test_dag_ai.py:28} INFO - LLM analysis completed successfully.
[2025-03-08T20:27:53.387+0000] {test_dag_ai.py:31} ERROR - LLM analysis failed: text
[2025-03-08T20:27:53.403+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/email.py:120: RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2025-03-08T20:27:53.404+0000] {email.py:229} INFO - Email alerting: attempt 1
[2025-03-08T20:27:55.006+0000] {email.py:241} INFO - Sent an alert email to ['monishassan1997@gmail.com']
[2025-03-08T20:27:56.856+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2025-03-08T20:27:56.941+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
